eve/tests/methods/delete.py

```diff
@@ -21,6 +21,27 @@ def test_unknown_resource(self):
         _, status = self.delete(url)
         self.assert404(status)
 
+    def test_bulk_delete_id_field(self):
```

Comment:
the new DELETE behavior should be explained in the docs, probably in the [Sub Resources](http://python-eve.org/features.html#sub-resources) of the features page.